File: matriz2d.cs
using System;

class Program
{
    static void Main()
    {
        int[][] TwoDimensionalArray = new int[][]
        {
            new int[] {1, 2, 3},
            new int[] {4, 5, 6},
            new int[] {7, 8, 9}
        };

        Console.WriteLine("Los elementos del array son: ");
        foreach (int[] row in TwoDimensionalArray)
        {
            foreach (int element in row)
            {
                Console.Write(element + " ");
                Console.WriteLine();
            }
        }
    }
}

File: matriz2d.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<std::vector<int>> TwoDimensionalArray = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };

    std::cout << "Los elementos del array son: \n";
    for (const auto& row : TwoDimensionalArray) {
        for (int element : row) {
            std::cout << element << " ";
            std::cout << "\n";
        }
    }

    return 0;
}